app/src/main/AndroidManifest.xml

```diff
@@ -1,10 +1,10 @@
 <?xml version="1.0" encoding="utf-8"?>
```

Comment:
@miPlodder , why you have made these changes in manifest, please revert them and just add a new <activity> tag for the activity that you have created.

Reply:
Ok